nevergrad/functions/pyomo/core.py

```diff
@@ -175,3 +201,71 @@ def _pyomo_constraint_wrapper(self, i: int, instru: tp.ArgsKwargs) -> bool:
             return ret
         else:
             raise NotImplementedError(f"Constraint type {self.all_constraints[i].ctype} is not supported yet.")
+
+
+# Simple Pyomo models, based on https://www.ima.umn.edu/materials/2017-2018.2/W8.21-25.17/26326/3_PyomoFundamentals.pdf.
+def get_pyomo_list():
```

Comment:
please include a docstring

Reply:
```suggestion
def generate_pyomo_functions() -> tp.Generator[base.Callable, None, None]:
```
the name was not informative: this is not even producint a list :s